src/main/java/io/stargate/sgv2/jsonapi/StargateJsonApi.java

```diff
@@ -337,7 +348,8 @@
                       """
                                 {
                                   "status": {
-                                      "deletedIds": ["1", "2"]
+                                      "deletedIds": ["1", "2"],
```

Comment:
I thought we were going to change `deletedIds` to `deletedCount`?

Reply:
Will change it. Missed to convert this to deletedCount.